CHANGELOG.md

```diff
@@ -30,10 +30,15 @@
  - [0.0.5](#005---20180602)
  - [0.0.4 and prior](#004-and-prior)
 
-## [2.0.0-alpha.1] - 2021/05/27
+## [2.0.0-alpha2.] - 2021/06/21
+
+TBD: this is the next release in the v2 code path, not yet released.
```

Comment:
Can you please remove TBD and add some concrete details in this section, i.e
migrate UDP Ingress to v1beta1
Add collision test cases

Reply:
Sounds good, changed in 3767ea8cd977002a1f592aed7f107bb79728ca31 :+1: